.github/workflows/ruby.yml

```
@@ -131,6 +131,26 @@ jobs:
       run: bundle exec rake ci
       env:
         ENGINE_CART_RAILS_OPTIONS: '--skip-git --skip-keeps --skip-action-cable --skip-test'
+  test_rails7_2:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby: [3.1, 3.2]
+    env:
+      RAILS_VERSION: 7.2.0
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby }}
+        bundler: 'default'
+    - name: Install dependencies
+      run: bundle install
+    - name: Run tests
+      run: bundle exec rake ci
+      env:
+        ENGINE_CART_RAILS_OPTIONS: '--skip-spring --skip-keeps --skip-test'
```

```suggestion
        ENGINE_CART_RAILS_OPTIONS: '--skip-keeps --skip-test'
```

`--skip-spring ` is not an option in 7.2